microprofile-7.0/README.adoc

```diff
@@ -0,0 +1,46 @@
+---
+title: "MicroProfile 7.0"
+date: 2024-06-19
+summary: "MicroProfile 7.0 Release Plan"
+---
+
+MicroProfile follows a train model for its Platform releases. Typically, three Platform releases are planned for each year in February, June, and October. MicroProfile releases components that are ready and have been approved for inclusion will be part of the Platform release. From this release onwards, Jakarta EE Core Profile will no longer be packaged in the MicroProfile BOM and the dependency has been made to be loosely coupled. MicroProfile Metrics specification is no longer included in the platform release, because the Metrics from MicroProfile Telemetry should be used instead. The specifications in MicroProfile 7.0 include:
+
+* Config 3.1
+* Fault Tolerance 5.0*
+* Health 4.0
+* JWT RBAC 3.0*
+* OpenAPI 4.0*
+* Rest Client 4.0*
+* Telemetry 2.0*
+
```

Comment:
To be consistent with other content, we should add "Jakarta EE 10 Core Profile". Especially when everyone is asking if MicroProfile 7.0 aligns with Jakarta EE 10 or 11.